catalog-info.yaml

```
@@ -55,3 +55,45 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-agent-package
+  description: Buildkite pipeline for packaging Elastic Agent package
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elastic-agent
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elastic-agent-package
+      description: Buildkite pipeline for packaging Elastic Agent package
+    spec:
+      repository: elastic/elastic-agent
+      pipeline_file: ".buildkite/pipeline.elastic-agent-package.yml"
+      provider_settings:
+        build_pull_request_forks: false
+        # 
+        build_pull_requests: true # requires filter_enabled and filter_condition settings as below when used with buildkite-pr-bot
```

As in the bot config `set_commit_status` is false
Should it be disabled here the status fo rcommits ? 
```suggestion
        build_pull_requests: true # requires filter_enabled and filter_condition settings as below when used with buildkite-pr-bot
        publish_commit_status: false # do not update status of commits for this pipeline
```